Macros/Compile_ACS.sas

```diff
@@ -125,20 +147,59 @@
     if inGeo;
     
     ** Create standard geography variable **;
-    
+
+	%if &geo = CITY %then %do;
+    length &geo_var $ 1;
+  
+    &geo_var = "1";
+  
+    label &geo_var = "&geo_label";
+  
+    format &geo_var &geo_format;
+	%end;
+
+
+	%if &geo = COUNTY %then %do;
+    length &geo_var $ 5;
+  
+    &geo_var = state || county ;
+  
+    label &geo_var = "&geo_label";
+  
+    format &geo_var &geo_format;
+	%end;
+
+	%if &geo = PLACE %then %do;
+    length &geo_var $ 7;
+  
+    &geo_var = state || place ;
+  
+    label &geo_var = "Regional places";
+  
+    format &geo_var &geo_format;
+	%end;
+
+
+    %else %do;
     length &geo_var $ &geo_length;
   
     &geo_var = substr( left( geoid ), 8 );
   
     label &geo_var = "&geo_label";
   
     format &geo_var &geo_format;
+	%end;
     
     ** Check for invalid geo variable values **;
+	** RP edited to only check for DC geographies **;
+
+	%if &state_ab = DC %then %do;
     
     if put( &geo_var, &geo_vformat ) = '' then do;
```

Comment:
Yes, you may be right. That code only gets executed for DC. But, we may want to change that now since we are actually summarizing data for MD and VA.